dea_waterbodies/make_time_series.py

```diff
@@ -165,28 +165,63 @@ def get_shapes(config_dict: dict,
 
 @click.command()
 @click.argument('ids', required=False, default='')
-@click.option('--config', '-c', type=click.Path(), default=None)
-@click.option('--shapefile', type=click.Path(), default=None)
-@click.option('--start', type=str, default=None)
-@click.option('--end', type=str, default=None)
-@click.option('--size', type=click.Choice(['ALL', 'SMALL', 'HUGE']),
-              default='ALL')
-@click.option('--missing-only/--not-missing-only', default=False)
-@click.option('--skip', type=click.Path(), default=None)
+@click.option('--config', '-c', type=click.Path(), default=None,
+              help='Path to a config.ini file. A config file can be provided '
+              'instead of using the command line options provided.')
+@click.option('--shapefile', type=click.Path(), default=None,
+              help='REQUIRED. Path to the waterbody polygon shapefile (or '
+              'geojson) you want to run the time series generation for.')
+@click.option('--output', type=click.Path(), default=None,
+              help='REQUIRED. Path to the output directory for the waterbody '
+              'timeseries.')
 @click.option('--time-span', type=click.Choice(['ALL', 'APPEND', 'CUSTOM']),
-              default='ALL')
-@click.option('--output', type=click.Path(), default=None)
+              default='ALL',
+              help='--time-span sets the time range for the waterbody '
```

Comment:
-> "Sets the time range..." The help will already print out --time-span by itself, so no need to include in the help string.